### PR TITLE
Background tasks fixes

### PR DESCRIPTION
**Pull request by terceiro**

### CODE REVIEW

**Comment by terceiro** [comment]
There. Reviewing commit-by-commit will be easier.

File: benchmarks/tasks.py [hunk 1]
```diff
@@ -24,6 +24,45 @@
 
 @celery_app.task(bind=True)
 def set_testjob_results(self, testjob):
+    try:
+        test_results = get_testjob_data(testjob)
+        store_testjob_data(testjob, test_results)
+    except testminer.LavaServerException as ex:
+        if ex.status_code == 503:
+            # server is too busy or in maintaince; bail out, will try again later
```
Comment (mwasilew) [inline benchmarks/tasks.py]: maybe we should have a log message at least?

Comment (terceiro) [inline benchmarks/tasks.py]: yes. I just pushed a few new commits addressing this

**Comment by mwasilew** [comment]
OK, let's merge it. I'm still a bit concerned about jenkins updates, but this isn't a mission critical feature.

**Pull request merged**